src/main/java/BasicMAPF/Solvers/PIBT/PIBT_Solver.java

```diff
@@ -430,6 +498,8 @@ protected void releaseMemory() {
         this.priorities = null;
         this.takenNodes = null;
         this.unhandledAgents = null;
+        this.constraints = null;
+        this.configurations = null;
```

Comment:
This should probably go in `master` too... I might end up just pulling this entire class into `master` to avoid them drifting too far apart...